.github/event-processor.config

```diff
@@ -22,5 +22,5 @@
   "IdentifyStalePullRequests": "On",
   "CloseAddressedIssues": "On",
   "LockClosedIssues": "On",
```

Comment:
Does this mean all closed issues (even manual ones) get locked?

Reply:
After a certain amount of time, yes